client/src/users/Users.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import axios from 'axios';
+
+class Users extends React.Component {
+  state = {
+    users: []
+  }
+
+  componentDidMount(){
+    const headers = {
+      authorization: localStorage.getItem('jwt')
+    };
+    const endpoint = 'http://localhost:9090/api/users/';
+    axios.get(endpoint, { headers })
+         .then(res => {
+           console.log(res.data);
+           this.setState({users: res.data});
+         })
+         .catch(err => {
+           console.log(err);
+         });
```

Comment:
Since you're just passing the parameter (`err`) along to another function, you could skip the extra step and use `console.log` as your callback:
```javascript
  .catch(console.log);
```